.github/ISSUE_TEMPLATE/organization-membership-request.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Organization Membership Request
+about: Request membership in an Istio Org
+
+---
+
+### GitHub Username
+e.g. (at)example_user
+
+### Company Affiliation or Individual
+e.g. Google, IBM, VMware, Red Hat or XX etc, otherwise, Individual
+
+### Requirements
+- Subscribed to istio-dev@googlegroups.com (https://groups.google.com/forum/#!forum/istio-dev).
+- Actively contributing to 1 or more areas.
+- Sponsored by 2 members.
+
+- [ ] I have reviewed the community membership guidelines (https://github.com/istio/community/blob/master/ROLES.md#member & https://github.com/istio/community/blob/master/CONTRIBUTING.md)
+- [ ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
+- [ ] I have subscribed to the istio-dev@googlegroups.com (https://groups.google.com/forum/#!forum/istio-dev) email list
+- [ ] I have made multiple contributions to the project or community. Contributions may include, but are not limited to:
```

Comment:
Would it be inappropriate to attach a number of contributions?

Should we also state a general breadth in the commits (i.e. more than just 10 dependency bumps)?

Reply:
@cmluciano imo its more about impact than raw stats.  For example, a new contributor submits a PR to the docs repo that enables delivery of helm charts via the website.  Huge impact, 1 commit.  That individual imo would be a prime candidate for membership once more work is done.  Whereas people fixing typos all over the code base with 30 commits - valued improvements but not necessarily impactful.

Can we can leave it up to the sponsors to decide?

Cheers
-steve